lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
@escowles Are you suggesting if I want to create an AF::File that points to a non-resolvable or non-HTTP URL then I might be better off storing that in another field (like `ebucore:locator`) and setting empty content instead of using `message/external-body` in the `mime_type`?

I figured it would be better to use the same approach to store the external reference no matter the protocol or resolvability and mark the `AF::File` as external content in all of those cases because that is what it is representing.